data/transition-sites/gds_alphagov_business_support_api.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: gds_alphagov_business_support_api
```

Comment:
Underscores in site names (and the filenames) are used by convention to [separate the organisation abbreviation from the site abbreviation](https://github.com/alphagov/transition-config#create-the-site-yaml-file). I'd suggest using a pattern like `gds_alphagov-business-support-api` for these sites to avoid confusion.